doc/developers.md

```diff
@@ -82,7 +82,29 @@ spdlog::set_level(spdlog::level::debug);
 spdlog::debug("Send help");
 ```
 
-## Where's the main ?
+### Compiling on UCL's myriad cluster
```

Comment:
Note that this should apply to all UCL RC clusters, not just Myriad, although since this program uses at most OpenMP other systems would be an overkill (for example Kathleen is for multi-node jobs only).

Myriad is usually spelled with capital letter in UCL RC docs
```suggestion
### Compiling on UCL's Myriad cluster
```